charts/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -355,3 +355,30 @@ serviceMonitor:
   labels: {}
   ## Here annotations can be added to the serviceMonitor
   annotations: {}
+
+## Optional containers for auth addons such as keycloak-gatekeeper
+optionalContainers:
+  ## Wether to enable dashboard-metrics-scraper
```

Comment:
Looks like copy paste leftover.

Reply:
It was an example of an optional container. Removed